### PR TITLE
Create mini datalake database

### PR DESCRIPTION
**Pull request by DJensen94**
Create models and handler to create mini datalake to store org and scan data

#  #

## 🗣 Description ##
This PR creates the models and code to generate the Mini Data Lake database, that will store data from the various teams and will eventually serve the Crossfeed application database data.

## 💭 Motivation and context ##
In order to expand Crossfeed's capabilities the database needs to be expanded to hold different vulnerability and scan types and hold data specific to each source, this new schema will allow us to house data from any source as it is expanded upon.

The schema will likely have some adjustments as time goes on and new sources are added.

## ✅ Pre-approval checklist ##

- [X] This PR has an informative and human-readable title.
- [X] Changes are limited to a single goal - *eschew scope creep!*
- [x] *All* future TODOs are captured in issues, which are referenced
      in code comments.
- [x] All relevant type-of-change labels have been added.
- [X] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [x] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [ ] Revert dependencies to default branches.
- [ ] Finalize version.

## ✅ Post-merge checklist ##

- [ ] Create a release.

### CODE REVIEW

File: backend/src/models/connection.ts [hunk 1]
```diff
@@ -13,11 +14,93 @@ import {
...
+  Contact,
+  DL_Cpe,
+  DL_Cve,
+  DL_Domain,
+  HostScan,
+  Host,
+  Ip,
+  Kev,
+  Location,
+  DL_Organization,
```
Comment (cduhn17) [inline backend/src/models/connection.ts]: Alphabetize?

File: backend/src/models/index.ts [hunk 2]
```diff
@@ -13,3 +13,29 @@ export * from './webpage';
 export * from './api-key';
 export * from './saved-search';
 export * from './organization-tag';
+
```
Comment (cduhn17) [inline backend/src/models/index.ts]: Remove blank space

**Review by cduhn17 — COMMENTED** [review]
See comments

File: backend/src/models/index.ts (hunk 2 above)
Comment (DJensen94) [reply backend/src/models/index.ts]: addressed in the following commit [0d8aa1a](https://github.com/cisagov/XFD/pull/110/commits/0d8aa1af615493043a5b8be96ccc688a9e3c7322)

File: backend/src/models/connection.ts (hunk 1 above)
Comment (DJensen94) [reply backend/src/models/connection.ts]: addressed in the following commit: [0d8aa1a](https://github.com/cisagov/XFD/pull/110/commits/0d8aa1af615493043a5b8be96ccc688a9e3c7322)

**Review by cduhn17 — APPROVED** [review]
Thanks for taking a look at those few comments.

**Review by aloftus23 — APPROVED** [review]
Looks good to me, just make sure you check everything in the pre-approval checklist

**Comment by schmelz21** [comment]
@dav3r or @jsf9k  - Did either of you want/can take a look at this one? We triggered a PR review with the .yml change.

**Review by jsf9k — APPROVED** [review]

**Pull request merged**